100.md

```diff
@@ -0,0 +1,528 @@
+        listOf("$plainText", "${hex_pub_key}", "${logged_in_user_npub}"),
+        null,
+        null,
+        null
+    )
+    ```
+  - result:
+    - Will return the **signature** column
+
+      ```kotlin
+        if (result == null) return
+
+        if (result.moveToFirst()) {
+            val index = it.getColumnIndex("signature")
+            val encryptedText = it.getString(index)
+        }
+      ```
+
+- **nip44_encrypt**
+  - params:
+
+    ```kotlin
+    val result = context.contentResolver.query(
+        Uri.parse("content://com.example.signer.NIP44_ENCRYPT"),
+        listOf("$plainText", "${hex_pub_key}", "${logged_in_user_npub}"),
+        null,
+        null,
+        null
+    )
+    ```
+  - result:
+    - Will return the **signature** column
+
+      ```kotlin
+        if (result == null) return
+
+        if (result.moveToFirst()) {
+            val index = it.getColumnIndex("signature")
+            val encryptedText = it.getString(index)
+        }
+      ```    
+
+- **nip04_decrypt**
+  - params:
+
+    ```kotlin
+    val result = context.contentResolver.query(
+        Uri.parse("content://com.example.signer.NIP04_DECRYPT"),
+        listOf("$encryptedText", "${hex_pub_key}", "${logged_in_user_npub}"),
+        null,
+        null,
+        null
+    )
+    ```
+  - result:
+    - Will return the **signature** column
+
+      ```kotlin
+        if (result == null) return
+
+        if (result.moveToFirst()) {
+            val index = it.getColumnIndex("signature")
+            val encryptedText = it.getString(index)
+        }
+      ```    
+
+- **nip44_decrypt**
+  - params:
+
+    ```kotlin
+    val result = context.contentResolver.query(
+        Uri.parse("content://com.example.signer.NIP44_DECRYPT"),
+        listOf("$encryptedText", "${hex_pub_key}", "${logged_in_user_npub}"),
+        null,
+        null,
+        null
+    )
+    ```
+  - result:
+    - Will return the **signature** column
+
+      ```kotlin
+        if (result == null) return
+
+        if (result.moveToFirst()) {
+            val index = it.getColumnIndex("signature")
+            val encryptedText = it.getString(index)
+        }
+      ```      
+
+- **decrypt_zap_event**
+  - params:
+
+    ```kotlin
+    val result = context.contentResolver.query(
+        Uri.parse("content://com.example.signer.DECRYPT_ZAP_EVENT"),
+        listOf("$eventJson", "", "${logged_in_user_npub}"),
+        null,
+        null,
+        null
+    )
+    ```
+  - result:
+    - Will return the **signature** column
+
+      ```kotlin
+        if (result == null) return
+
+        if (result.moveToFirst()) {
+            val index = it.getColumnIndex("signature")
+            val eventJson = it.getString(index)
+        }
+      ```    
+
+# Usage for Web Applications
+
+Since web applications can't receive a result from the intent you should add a modal to paste the signature or the event json or create a callback url.
```

Comment:
```suggestion
Since web applications can't receive a result from the intent, you should add a modal to paste the signature or the event json or create a callback url.
```